base_import_pdf_by_template/models/baseimportpdfbytemplate_mixin.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class BaseimportpdfbytemplatetMixin(models.AbstractModel):
+    _name = "baseimportpdfbytemplate.mixin"
+    _description = "Functions for baseimportpdfbytemplate mixin documents"
+
+    def _import_record_base_import_pdf_by_template(self, record, attachment):
```

Comment:
I don't think this little code is worth for a whole mixin

Reply:
The idea of this was that it could be used in another context, for example, when processing fetchmail and creating a purchase/sale order.

Reply:
That's not a real case right now (there are no such mechanisms) and it's very few code to be reused, so for now, please don't do it.